CHANGES.txt

```diff
@@ -107,6 +107,8 @@ Changes
 Fixes
 =====
 
+- Fix an exception which occurred when using row ids in IN or ANY queries.
```

Comment:
our common way is to write **Fixed**

Reply:
"row ids" may not be so clear what it is.
maybe say: ... when using the `_id` system column in `IN` and `ANY` queries. just a suggestion.